client/components/RelatedProducts/Outfit/CustomOutfitList.jsx

```diff
@@ -16,7 +17,7 @@ const CustomOutfitList = ({
       <Carousel showEmptySlots itemsToShow={4}>
         <BlankOutfitCard addToOutfit={addToOutfit} />
           {outfitCardsData.length ? (
-            outfitCardsData.map(card => <OutfitProductCard key={card.style_id} outfitCardData={card} />)
+            outfitCardsData.map(card => <OutfitProductCard key={card.style_id} outfitCardData={card} removeCard={removeCard}/>)
             ) : (
             <> </>
```

Comment:
I did a ternary the same way, where I render a react fragment, if I want to render nothing. I don't know if this is best practice it's just sorta something that worked. I'm going to keep an eye out for better solutions, although this is pretty clean and understandable in my opinion.